WireDomain/Frameworks/README.txt

```diff
@@ -0,0 +1,3 @@
+https://github.com/wireapp/kalium/blob/develop/backup/README.md
+./gradlew :backup:assembleBackupReleaseXCFramework
+(./gradlew :backup:assembleBackupDebugXCFramework)
```

Comment:
The commands to build the framework. This will be deleted once the binary is deployed.